apps/web/src/hooks/useTransferFee.ts

```diff
@@ -1,15 +1,24 @@
 import BigNumber from "bignumber.js";
 import { useNetworkContext } from "@contexts/NetworkContext";
+import { Network } from "types";
+import { ETHEREUM_SYMBOL } from "../constants";
 
 /**
  * Computes transfer fee
  * Any changes to the fee logic can be updated here
  */
-const FEE_PERCENTAGE = 0.003;
+
+const EVM_TO_DFC_FEE_PERCENTAGE = 0.01;
+const DFC_TO_EVM_FEE_PERCENTAGE = 0.03;
```

Comment:
```suggestion
const EVM_TO_DFC_FEE_PERCENTAGE = 0.001;
const DFC_TO_EVM_FEE_PERCENTAGE = 0.003;
```
it's 0.0x% right? so it should be this